### PR TITLE
[WIP] feature/electron-security

### PR DESCRIPTION
**Pull request by gempi**
- [x] Only load secure content
- [x] Disable the Node.js integration in all renderers that display remote content
- [ ] ~~Enable context isolation in all renderers that display remote content~~
- [x] Use ses.setPermissionRequestHandler() in all sessions that load remote content (No permisson requests)
- [x] Do not disable webSecurity
- Define a Content-Security-Policy and use restrictive rules (i.e. script-src 'self')
  - [x] Define a Content Security Policy (not required)
  - [x] CSP HTTP Header
  - [x] CSP Meta Tag
- [x] Override and disable eval, which allows strings to be executed as code.
- [x] Do not set allowRunningInsecureContent to true
- [x] Do not enable experimental features
- [x] Do not use enableBlinkFeatures
- [x] WebViews: Do not use allowpopups
- [ ] WebViews: Verify the options and params of all <webview> tags **(related to nodeIntegration)**

### CODE REVIEW

**Comment by gempi** [comment]
@schmidsi waiting for custom event handler to disable nodeIntegration

**Comment by fubhy** [comment]
Disabled node integration, but we can't use context isolation. Not possible. Otherwise we can't use IPC.

**Comment by fubhy** [comment]
And sorry, I worked on this in a different branch. My bad. I didn't see this PR before.

**Comment by schmidsi** [comment]
@gempi Please rebase to the master so that we can merge this branch.

Furthermore, ensure the following:
- [ ] A click on a link inside electron does open the OS standard browser and is not followed internally
- [ ] Drag & drop of files is not possible
- [ ] It is not possible for a script to load external resources (please try this to verify). No fonts, css, js, img can be loaded from www

File: packages/manager-interface/src/pages/_document.tsx [hunk 1]
```diff
@@ -16,7 +16,10 @@ export default class MyDocument extends Document {
             content="width=device-width, initial-scale=1, shrink-to-fit=no"
           />
           <meta name="theme-color" content="#000000" />
-          <meta http-equiv="Content-Security-Policy" content="default-src 'none'">
+          <meta
+            http-equiv="Content-Security-Policy"
+            content="default-src 'self' 'unsafe-inline'; connect-src https://kovan.melonport.com kovan.melonport.com kovan.melonport.com:443; font-src data: file:;"
```
Comment (schmidsi) [inline packages/manager-interface/src/pages/_document.tsx]: Actually the electron app should not connect to hosted nodes. Only local ones: `http://localhost:8545`

Comment (gempi) [reply packages/manager-interface/src/pages/_document.tsx]: @schmidsi are there some other urls which we have to allow?

Comment (schmidsi) [reply packages/manager-interface/src/pages/_document.tsx]: @gempi Nope, for maximum security we should lock down the whole thing to only communicate with the local JSON-RPC interface.

**Pull request merged**